application/src/bll/reducers/paymentCardReducer.ts

```diff
@@ -54,4 +54,30 @@ export const setCustomerData = (
     total
   } as const);
 
+export const saveDataOrders = (
+  orderId: string,
+  surname: string,
+  name: string,
+  mobile: string,
+  totalCartSum: number
+) => {
+  const email = localStorage.getItem('email');
+  const time = new Date();
+  const day = time.getUTCDate();
+  const month = time.getMonth() + 1;
+  const year = time.getFullYear();
+  const date = `${day}.${month}.${year}`;
```

Comment:
Let's create common util function which will be responsible for data treatment

Reply:
I created function getDate(total:number){...}